RealmBrowser/Controllers/RLMInstanceTableViewController.m

```diff
@@ -360,14 +353,14 @@ -(NSView *)tableView:(NSTableView *)tableView viewForTableColumn:(NSTableColumn
             RLMNumberTableCellView *numberCellView = [tableView makeViewWithIdentifier:reuseIdentifier owner:self];
             if (!numberCellView) {
                 numberCellView = [RLMNumberTableCellView viewWithIdentifier:reuseIdentifier];
-                numberCellView.textField.delegate = self;
                 numberCellView.textField.target = self;
                 numberCellView.textField.action = @selector(editedTextField:);
             }
 
-            numberCellView.textField.stringValue = [realmDescriptions printablePropertyValue:propertyValue ofType:type];
-
-            ((RLMNumberTextField *)numberCellView.textField).number = propertyValue;
+            // Lets move all formatting stuff to cell itself
+            // numberCellView.textField.stringValue = [realmDescriptions printablePropertyValue:propertyValue ofType:type];
```

Comment:
these comments don't really add value unless you know what we were doing before, which is what git history is for 😉